CONTRIBUTING.md

```diff
@@ -32,15 +28,7 @@ If you'd like to get involved in AdoptOpenJDK then we recommend you read the [Kn
 
 ## Joining the GitHub Org and Teams
 
-Please raise an [openjdk-TSC issue](https://github.com/AdoptOpenJDK/TSC/issues) if you'd like to be added to the 
-AdoptOpenJDK GitHub organisation.
-
-There are some specialist teams within the org:
- 
-1. Please raise an [openjdk-infrastructure issue](https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues) if you'd like to be added to the 
-infrastructure team.
-1. Please raise an [openjdk-TSC issue](https://github.com/AdoptOpenJDK/openjdk-TSC/issues) if you'd like to be added to the 
-JCK team.
+Please raise an [openjdk-TSC issue](https://github.com/AdoptOpenJDK/TSC/issues) if you'd like to be added to the AdoptOpenJDK GitHub organisation and you'll be directed to the appropriate team(s).
```

Comment:
```suggestion
Please raise an [TSC issue](https://github.com/AdoptOpenJDK/TSC/issues) if you'd like to be added to the AdoptOpenJDK GitHub organisation and you'll be directed to the appropriate team(s).
```
The use of `openjdk-TSC` here is misleading since we don't have a repository by that name